pages/open_cas_linux/guide_running.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Open CAS Linux - Admin Guide
+last_updated: April 29, 2019
+toc: true
+permalink: guide_running.html
+---
+
+Running Open CAS Linux
+==================
+
+This chapter describes how to perform typical administrative activities with
+Open CAS Linux.
+
+Initial configuration of Cache instances
+----------------------------------------
+
+Once Open CAS Linux is installed, but no cache instances are yet configured and
+running, the user can easily configure and start cache instances. Refer to
+guidance in section 4.1 for *utils/opencas.conf* configuration, then as
+follows:
+
+1. Edit and configure caches and cores as desired via the
+*opencas.conf* file.
+
+2. Execute the following:
+
+>   \# utils/casctl init
+
+Open CAS Linux will read contents of the *opencas.conf* file, initialize the
+cache/core pairings, and place caches/core devices into an operational or active
+status.
+
+3. If the cache device contains data, partitions, or a file system from
+prior use, it will not normally initialize. If you receive a message during
+initialization about an existing file system, it may need the --force option
+to initialize. Be aware that the --force option will destroy any data or
+formatting on the device. Caches will be started regardless of existing data
+or partitions. Execute the following:
+
+>   \# intelcas init --force
```

Comment:
intelcas -> casctl (here and in several places below)

Reply:
I will fix this